pages.ko/common/docker-build.md

```diff
@@ -0,0 +1,32 @@
+# docker build
+
+> 도커파일로부터 이미지 빌드.
+> 더 많은 정보: <https://docs.docker.com/engine/reference/commandline/build/>.
+
+- 현재 디렉토리 안의 도커파일을 이용해 도커 이미지 빌드:
+
+`docker build .`
+
+- 명시된 URL의 도커파일로부터 도커 이미지 빌드:
+
+`docker build {{github.com/creack/docker-firefox}}`
+
+- 도커 이미지 빌드 및 태그 추가:
+
+`docker build --tag {{name:tag}} .`
```

Comment:
Please translate the placeholders, like `name`, `tag`, etc., for get this easier for non-English speakers.

Reply:
I just edited it. Thanks.